testsuite/unittests/test_rate_connections.sli

```diff
@@ -0,0 +1,250 @@
+/*
+ *  test_rate_connections.sli
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+  /* BeginDocumentation
+
+    Name: testsuite::test_rate_connections - This test ensures that rate connections (rate_connection, delay_rate_connection, diffusion_connection) work properly.
+
+    Synopsis: (test_rate_connections) run -> NEST exits if test fails
+
+    Description:
+    The test ensures that legal rate connections can be created and that an exception 
+    is thrown if one tries to create illegal rate connections. 
+    Furthermore it is checked that the delay cannot be set for rate_connection connections and
+    that delay and weight cannot be set for diffusion_connection connections.
+
+    Author: Jan Hahne
+    SeeAlso: rate_connection, delay_rate_connection, diffusion_connection
+  */
+
+(unittest) run
+/unittest using
+
+% The following test needs the model siegert_neuron, so
+% this test should only run if we have GSL
+skip_if_without_gsl
+
+M_ERROR setverbosity
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Checks for rate_connection %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+/supported
+[ /lin_rate_ipn 
+  /lin_rate_opn
+  /thresholdlin_rate_ipn
+  /thresholdlin_rate_opn
+  /tanh_rate_ipn
+  /tanh_rate_opn ]
+def
+
+/not_supported
+[ /hh_psc_alpha_gap
+  /siegert_neuron
+  /iaf_psc_neuron]
```

Comment:
`iaf_psc_neuron` is not a known model name. The test runs through but I suspect, that the part where you use /iaf_psc_neuron actually fails with `UnknownModelName` instead of throwing an error because of the synapse. 
I assume you just want to test one representative spiking neuron here, so I suggest changing this to 'iaf_psc_delta'.
The same for the other parts below.